From ba370cdb895ee952a795e4934ef0b51fbbffe720 Mon Sep 17 00:00:00 2001
From: Mike Yuan <me@yhndnzj.com>
Date: Sat, 11 Jan 2025 18:38:49 +0100
Subject: [PATCH] core/manager: drop duplicate bus track deserialization

bus_init_api() now does this internally
(after 8402ca04d1a063c3d8a9e3d5c16df8bb8778ae98).

(cherry picked from commit af0e10354e567bfd0b9521376b2aad55f12a4e3d)

Related: RHEL-75081
---
 src/core/manager.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/core/manager.c b/src/core/manager.c
index bee94fb80d..5212650fec 100644
--- a/src/core/manager.c
+++ b/src/core/manager.c
@@ -1658,10 +1658,6 @@ int manager_startup(Manager *m, FILE *serialization, FDSet *fds) {
         /* Connect to the bus if we are good for it */
         manager_setup_bus(m);
 
-        /* Now that we are connected to all possible busses, let's deserialize who is tracking us. */
-        (void) bus_track_coldplug(m->api_bus, &m->subscribed, false, m->subscribed_as_strv);
-        m->subscribed_as_strv = strv_free(m->subscribed_as_strv);
-
         /* Third, fire things up! */
         manager_coldplug(m);
 
